src/Microsoft.ML.Transforms/Dracula/CountTableTransformer.cs

```diff
@@ -0,0 +1,734 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.Data.IO;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Transforms;
+
+[assembly: LoadableClass(CountTableTransformer.Summary, typeof(IDataTransform), typeof(CountTableTransformer), typeof(CountTableTransformer.Options), typeof(SignatureDataTransform),
+    CountTableTransformer.UserName, "CountTableTransform", "CountTable", "Count")]
+
+[assembly: LoadableClass(CountTableTransformer.Summary, typeof(IDataTransform), typeof(CountTableTransformer), null, typeof(SignatureLoadDataTransform),
+    CountTableTransformer.UserName, CountTableTransformer.LoaderSignature)]
+
+[assembly: LoadableClass(CountTableTransformer.Summary, typeof(CountTableTransformer), null, typeof(SignatureLoadModel),
+    CountTableTransformer.UserName, CountTableTransformer.LoaderSignature)]
+
+[assembly: LoadableClass(typeof(IRowMapper), typeof(CountTableTransformer), null, typeof(SignatureLoadRowMapper),
+   CountTableTransformer.UserName, CountTableTransformer.LoaderSignature)]
+
+[assembly: EntryPointModule(typeof(CountTable))]
+
+namespace Microsoft.ML.Transforms
+{
+    public sealed class CountTableEstimator : IEstimator<CountTableTransformer>
```

Comment:
Does this class need to be `public`? How can an external user use this class?

Reply:
Also, xml comments on all public surface area.

---
In reply to: [356307263](https://github.com/dotnet/machinelearning/pull/4514#discussion_r356307263) [](ancestors = 356307263)